flake.nix

```diff
@@ -75,6 +75,9 @@
                 pkg-config
                 file
                 ccls
+                sphinx
+                sphinx-autobuild
+                #sphinx-rtd-theme
```

Comment:
Can't workout why this doesn't work. Not having it is still better than nothing since we can use the livedoc makefile target.